go.mod

```diff
@@ -208,3 +208,7 @@ replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-d
 go 1.21.4
 
 retract v0.0.2
+
+replace github.com/btcsuite/btcd => github.com/yyforyongyu/btcd v0.21.0-beta.0.20240702184211-31cf2383abd1
+
+replace github.com/btcsuite/btcwallet => github.com/yyforyongyu/btcwallet v0.16.11-0.20240702184603-cf50b410fcb1
```

Comment:
I don't think we want to replace btcsuite with your fork.

Reply:
😈